src/Dapi/Response/CreateACHPullResponse.cs

```diff
@@ -0,0 +1,23 @@
+using Dapi.Types;
+using Newtonsoft.Json;
+
+namespace Dapi.Response {
+    public class CreateACHPullResponse : BaseResponse {
+
+        /// <summary>
+        /// This is used only to automate the deserialization of the get response.
+        /// This is a private constructor to this lib.
+        /// </summary>
+        [JsonConstructor]
+        internal CreateACHPullResponse(APIStatus status, bool success, string operationID) :
```

Comment:
This constructor is missing the `userInputs`, `type` and `msg` fields.

Reply:
7e230912b2e8d8a02458e86471dd2bd1f8e5ff2d